docs/source/commands.rst

```diff
@@ -67,24 +67,20 @@ optional arguments:
 stop
 ====
 
-This first sets the Auto Scaling Group parameters to :code:`min/max/desired =
-0/0/0` then stops the Master Server. This polls on the status of the master
-server until it is stopped. If desired, one can optionally use the
-:code:`--compute-only argument` to leave the Master Server intact.
+Sets the Auto Scaling Group parameters to :code:`min/max/desired = 0/0/0`
 
-.. note:: A stopped cluster won't charge for EC2 usage but will still charge for
-   EBS usage and Elastic IP addresses. Each time you bring up an instance it
-charges you for an hour so bringing it up and down multiple times within an hour
-isn't reccomended. For more info see `Stop and Start Your Instance
-<https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Stop_Start.html>`_.
+.. note:: A stopped cluster will only stops complete-fleet, but Master server

```

Comment:
Just so this statement isn't ambiguous, we should say stopping a cluster will only _terminate_ the compute fleet.

Reply:
done